checkstyle-suppressions.xml

```diff
@@ -4,4 +4,5 @@
     <suppress files="generated-sources" checks=".*"/>
     <suppress files="generated-test-sources" checks=".*"/>
     <suppress files="com.cerner.beadledom.client.resteasy.ApacheHttpClient4Dot3Engine" checks=".*"/>
+    <suppress files="com.cerner.beadledom.jaxrs.PATCH" checks="AbbreviationAsWordInName" />
```

Comment:
I just want to confirm this is necessary because checkstyle is expecting the classname to be `Patch`?

If JAX-RS hadn't already set the standard for these annotations we would name it that way, but `PATCH` is correct in following the standard of the other JAX-RS annotations.